app/controllers/users_controller.rb

```diff
@@ -1,6 +1,17 @@
 class UsersController < ApplicationController
   skip_before_action :require_login
 
+  def getList
```

Comment:
@JuanMaGiraldo if you place this code fragment in a module and then include it ... will improve your code.

Reply:
Advice received and applied, thank you!